lib/ruby_lsp/requests/document_symbol.rb

```diff
@@ -206,16 +213,22 @@ def on_var_field(node)
         params(
           name: String,
           kind: Integer,
-          range_node: SyntaxTree::Node,
-          selection_range_node: SyntaxTree::Node,
+          range_node: YARP::Node,
+          selection_range_node: T.any(YARP::Node, YARP::Location),
```

Comment:
Is it slightly innaccurate to call this a `_node` since `YARP::Location` isn't a node? 🤔

Reply:
Yes. I pushed a commit that switches to using locations, which is what we want with YARP now that some nodes no longer exist.